kogito-editors-js/packages/boxed-expression-component/src/components/Resizer/Resizer.tsx

```diff
@@ -76,6 +76,7 @@ export const Resizer: React.FunctionComponent<ResizerProps> = ({
       onHorizontalResizeStop?.(width);
     }
 
+    // TODO: Remove global listener
```

Comment:
Great catch. We should avoid using global listeners whenever possible. I understand that this is out of scope for this PR, but kudos for boy-scouting. Could you please create a Jira for this issue and link the Jira number on this comment.

Reply:
Ok! I'm going to create a JIRA to track this.

Reply:
Actually, I've manage to fix the issue already.